dev-resources/template/2_lrs.yml

```diff
@@ -50,10 +50,12 @@ Parameters:
   # Lambda Script
   S3Bucket:
     Type: String
-    Default: lrsql-demo
+    Description: S3 Bucket used to source the DB Init function. The default is hosted publicly so you probably don't need to change this.
+    Default: lrsql-dbfn
```

Comment:
Is there a reason for the S3 bucket name change?

Reply:
It's now deployed via template, and I wanted a more specific/relevant name

Reply:
And so that "dbfn" is the template? Trying to see how it's a more relevant name.